pkgs/applications/science/electronics/oregano/default.nix

```
@@ -0,0 +1,74 @@
+{ fetchFromGitHub
+, glib
+, goocanvas2
+, gnucap
+, groff
+, gtk3
+, gtksourceview
+, intltool
+, libxml2
+, lib
+, makeWrapper
+, ngspice
+, perl
+, pkg-config
+, python3
```

```suggestion
, python310
```
Doesn't work with later versions.